cli/tests/pcluster/configure/test_pcluster_configure.py

```diff
@@ -352,7 +400,17 @@ def test_with_input_no_automation_no_errors_with_config_file(mocker, capsys, tes
 
     MockHandler(mocker)
 
-    _run_input_test_with_config(mocker, config, old_config_file, error, output, capsys, with_input=True)
+    _run_input_test_with_config(
+        mocker,
+        config,
+        old_config_file,
+        error,
+        output,
+        capsys,
+        with_input=True,
+        master_instance="m6g.xlarge",
```

Comment:
What happens if user specifies a mix of ARM and X86 instances for head node and compute? I guess `pcluster configure` will succeed and config validation when creating cluster will fail right?

Reply:
Yeah. I've added a backlog item to eventually validate the config produced in the same way we do when creating a cluster, because as you said this validation logic is already defined there. Reasons it's not being enabled right now:
* The VPC automation doesn't take into account what instance types a user wants. This is fine for t2's, but for rarer instance types we probably need to query for availability as best we can and then create VPCs and subnets accordingly. Without that, the dry run in the config validation would fail frequently.
* We need to implement a way for the derived parameters to be updated when one of the parameters on which they're dependent changes during the `pcluster configure` process.

Reply:
https://github.com/aws/aws-parallelcluster/pull/1915#discussion_r463659994